.github/workflows/deployment-smoke-tests.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         env:
           CI: true
           PLAYWRIGHT_BASEURL: ${{ secrets.BASE_URL }}
-        run: npx playwright test deployment-tests/*
+        run: npx playwright test deployment-tests/* --project=cross-browser
```

Comment:
we should pick a single browser to run deployment tests with (and only install that one browser in pipeline)

Reply:
Perhaps this would all be configured in a dedicated "deployment" project